lms/djangoapps/courseware/views.py

```diff
@@ -726,6 +726,23 @@ def registered_for_course(course, user):
         return False
 
 
+def get_cosmetic_display_price(course, registration_price):
+    """
+    Return Course Price as a string preceded by correct currency, or 'Free'
+    """
+    currency_symbol = settings.PAID_COURSE_REGISTRATION_CURRENCY[1]
+
+    price = course.cosmetic_display_price
+    if registration_price > 0:
+        price = registration_price
+
+    if price:
+        return "{}{}".format(currency_symbol, price)
```

Comment:
Since this will more likely get used by microsites and open installations, and not edx.org, and since there is international variation in how this is displayed, I recommend internationalizing such as

```python
_("{currency_symbol}{price}").format(...)
```

I would also add in a translator's comment as to what this might look like (`"This may look like $50 or ¥900"`, vs `"This may look like 50 USD or 900 yen"`). I say this because there is a place in the codebase that's similar but uses "USD" or other abbreviations.